airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/InvalidCursorException.java

```diff
@@ -0,0 +1,26 @@
+package io.airbyte.integrations.source.relationaldb;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class InvalidCursorException extends RuntimeException {
+
+  public InvalidCursorException(final List<InvalidCursorInfo> tablesWithInvalidCursor) {
+    super("The following tables have invalid columns selected as cursor, please select a valid column as a cursor. " + tablesWithInvalidCursor.stream().map(InvalidCursorInfo::toString)
```

Comment:
nitpick: maybe replace "a valid column" with "a column with a well-defined ordering"? So that it's clear why the cursor isn't valid